Core/Node-API/CMakeLists.txt

```diff
@@ -165,7 +165,8 @@ add_library(napi ${SOURCES})
 target_compile_definitions(napi
     PUBLIC NODE_ADDON_API_DISABLE_DEPRECATED
     PUBLIC NODE_ADDON_API_DISABLE_NODE_SPECIFIC
-    PUBLIC NAPI_VERSION=3
+    PUBLIC NAPI_VERSION=5
```

Comment:
I'm guessing we don't have these `NAPI_VERSION >= 5` functions implemented for all platforms?

```c++
// Dates
NAPI_EXTERN napi_status NAPI_CDECL napi_create_date(napi_env env,
                                                    double time,
                                                    napi_value* result);

NAPI_EXTERN napi_status NAPI_CDECL napi_is_date(napi_env env,
                                                napi_value value,
                                                bool* is_date);

NAPI_EXTERN napi_status NAPI_CDECL napi_get_date_value(napi_env env,
                                                       napi_value value,
                                                       double* result);

// Add finalizer for pointer
NAPI_EXTERN napi_status NAPI_CDECL napi_add_finalizer(napi_env env,
                                                      napi_value js_object,
                                                      void* finalize_data,
                                                      napi_finalize finalize_cb,
                                                      void* finalize_hint,
                                                      napi_ref* result);
```

Can you file an issue to figure this out?

Reply:
Correct! https://github.com/BabylonJS/JsRuntimeHost/issues/81